plugins/webhooks/index.js

```diff
@@ -83,7 +83,7 @@ function startPRJob(options, request) {
             });
         })
         // create a build
-        .then(event => buildFactory.create({ jobId, sha, username, eventId: event.id }))
+        .then(event => buildFactory.create({ jobId, sha, username, eventId: event.id, prRef }))
```

Comment:
We're not going to save `prRef` to the table, we just want to use it in models to call `getCheckoutCommand`